src/main/java/cohen/morseCode/MorseCode.java

```diff
@@ -0,0 +1,74 @@
+package cohen.morseCode;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class MorseCode
+{
+    private final ArrayList<String> english = new ArrayList<>();
```

Comment:
Take a look at `List.of()`
https://docs.oracle.com/javase/9/docs/api/java/util/List.html#of-E...-